deployment/settings/services/imagetag.setting

```diff
@@ -1,4 +1,4 @@
-MINDARO_DEVHOSTAGENT_TAG=1.1.0
-MINDARO_DEVHOSTAGENT_RESTORATIONJOB_TAG=1.2.0
-MINDARO_ROUTINGMANAGER_TAG=1.1.0
+MINDARO_DEVHOSTAGENT_TAG=1.2.0
```

Comment:
@gambtho updated this so that B2K picks up new images or else existing k8's cluster will reuse the images.